yarn.lock

```diff
@@ -3105,6 +3124,20 @@
     fflate "^0.4.4"
     mitt "^3.0.0"
 
+"@sentry-internal/rrweb@2.17.0":
```

Comment:
looking like duplicate packages here

you may need to run `npx yarn-deduplicate && yarn install`

Reply:
just ran - does it look correct now @scttcper ?

Reply:
still looks like duplciates, it might be because you haven't updated the other two packages

```    
"@sentry-internal/rrweb-player": "2.12.0",
"@sentry-internal/rrweb-snapshot": "2.12.0",
```

Reply:
just updated